lib/rules/require-capitalized-comments.js

```diff
@@ -1,7 +1,13 @@
 var assert = require('assert');
+var regenerate = require('regenerate');
+var Lu = require('unicode-6.3.0/categories/Lu/code-points');
+var Ll = require('unicode-6.3.0/categories/Ll/code-points');
+var Lt = require('unicode-6.3.0/categories/Lt/code-points');
+var Lm = require('unicode-6.3.0/categories/Lm/code-points');
+var Lo = require('unicode-6.3.0/categories/Lo/code-points');
 
-var upperCasePattern = /[A-Z]/;
-var alphabeticalPattern = /[a-z|A-Z]/;
+var letterPattern = regenerate(Lu, Ll, Lt, Lm, Lo).toRegExp();
```

Comment:
Instead of including `Lu`, `Ll`, `Lt`, `Lm`, and `Lo` separately, why not just use [`L`](http://unicode.org/reports/tr44/#GC_Values_Table)?